### PR TITLE
PIM-1793: Adding script to alter total bales column to 6 digits

### PR DESCRIPTION
**Pull request by mhostett**

### CODE REVIEW

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+--add two more digits to the total bales columns
+
+ALTER TABLE cuws.declared_yield_field_rollup_forage ALTER COLUMN total_bales_loads TYPE numeric(6,0);
```
Comment (dpitk2222) [inline cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql:3]: I noticed that declared_yield_contract_cmdty_forage also has a total_bales_loads column. Do you think it should be expanded too?

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/cirras.cuws.03_05_00_01_fix.ddl.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+{
+  "databaseChangeLog": [
+{ "changeSet": {"author":"mhostettler","id":"86","changes": [
```
Comment (dpitk2222) [inline cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/cirras.cuws.03_05_00_01_fix.ddl.json:4]: id 86 is also used by the dml master script. So probably the dml master script should use id 87?

**Review by dpitk2222 — CHANGES_REQUESTED** [review]

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql (hunk 1 above)
Comment (mhostett) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql:3]: We should do that but in another release

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/cirras.cuws.03_05_00_01_fix.ddl.json (hunk 2 above)
Comment (mhostett) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/cirras.cuws.03_05_00_01_fix.ddl.json]: Updated the id of this script to 87

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql (hunk 1 above)
Comment (dpitk2222) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql:3]: We'll expand it later, if needed.

**Review by dpitk2222 — APPROVED** [review]

**Pull request merged**